### DISCUSSION TITLE
Add support for pinned chats

### DISCUSSION THREAD

**Discussion by arcxne** [d99kris/nchat#168] [Ideas]
In Whatsapp, we have the ability to pin chats such that they appear at the top of all chats. It'd be nice to have this feature so that I can easily access my most important chats.

**Comment by d99kris** [comment]
Hi @arcxne - support for honoring pinned status in the chat list has been added in e8143b3. 

May I check if this feature request was also for being able to pin/unpin chats from nchat?  
If yes, I'll leave this request open. Thanks!

**Reply by arcxne** [reply]
I wouldn't really bother with pinning/unpinning chats in nchat itself, so no problem! Thanks for your time and effort!

**Reply by d99kris** [reply]
Although not requested in this feature request, I just wanted to mention that support for pinning / unpinning chats has been added now in `v5.14.18` / 7ec3aa2.
